### PR TITLE
Backport some stuff to LSP3

### PR DESCRIPTION
**Pull request by rchl**
I want to use those in lsp_utils

### CODE REVIEW

File: plugin/core/handlers.py [hunk 1]
```diff
@@ -20,6 +21,14 @@ def config(self) -> ClientConfig:
     def additional_variables(cls) -> Optional[Dict[str, str]]:
         return None
 
+    @classmethod
+    def storage_path(cls) -> str:
+        """
+        The storage path. Use this as your base directory to install server files. Its path is '$DATA/Package Storage'.
+        You should have an additional subdirectory preferrably the same name as your plugin.
```
Comment (rwols) [inline plugin/core/handlers.py:28]: s/plugin/handler/g

File: plugin/core/views.py [hunk 2]
```diff
@@ -27,8 +27,21 @@ def get_line(window: Optional[sublime.Window], file_name: str, row: int) -> str:
         return linecache.getline(file_name, row + 1).strip()
 
 
+def get_storage_path() -> str:
+    """
+    The "Package Storage" is a way to store server data without influencing the behavior of Sublime Text's "catalog".
+    Its path is '$DATA/Package Storage', where $DATA means:
+
+    - on macOS: ~/Library/Application Support/Sublime Text
```
Comment (rwols) [inline plugin/core/views.py:35]: Sublime Text 3

File: plugin/core/views.py [hunk 3]
```diff
@@ -27,8 +27,21 @@ def get_line(window: Optional[sublime.Window], file_name: str, row: int) -> str:
...
+    - on Windows: %AppData%/Sublime Text/Roaming
```
Comment (rwols) [inline plugin/core/views.py:36]: Sublime Text 3

File: plugin/core/views.py [hunk 4]
```diff
@@ -27,8 +27,21 @@ def get_line(window: Optional[sublime.Window], file_name: str, row: int) -> str:
...
+    - on Linux: $XDG_CONFIG_DIR/sublime-text
```
Comment (rwols) [inline plugin/core/views.py:37]: sublime-text-3 :)

**Review by rwols — APPROVED** [review]

File: plugin/core/handlers.py (hunk 1 above)
Comment (rchl) [reply plugin/core/handlers.py:28]: I think technically the most correct term should be "package" but it's really unimportant.

**Pull request merged**